src/app/core/sign-in/sign-in.service.ts

```diff
@@ -12,29 +12,38 @@ import { getOrcidNumber } from '../../constants'
   providedIn: 'root',
 })
 export class SignInService {
+  private headers: HttpHeaders
+
   constructor(
     private _http: HttpClient,
     private _errorHandler: ErrorHandlerService
-  ) {}
-
-  signIn(data) {
-    const headers = new HttpHeaders().set(
+  ) {
+    this.headers = new HttpHeaders().set(
       'Content-Type',
       'application/x-www-form-urlencoded;charset=utf-8'
     )
+  }
+
+  signIn(SingInLocal) {
```

Comment:
Add the new type here so this function gets the definition

Reply:
Done